### PR TITLE
community/mpv: enable wayland

### PR DESCRIPTION
**Pull request by tanertas**
main/libva(-glx): upgrade to 2.4.0, enable wayland
main/libva-intel-driver: upgrade to 2.3.0, enable wayland

### CODE REVIEW

**Comment by andypost** [comment]
Probably fixed with 198e5582

**Comment by tanertas** [comment]
> Probably fixed with [198e558](https://github.com/alpinelinux/aports/commit/198e5582eff680a73286d15d300040902cd54e2b)

I have updated this PR accordingly.

**Comment by tanertas** [comment]
@ncopa @Ikke can anyone take a look at this?

**Comment by Ikke** [comment]
@tanertas Could you rebase this onto the latest master? Then we will get drone CI as well. Thanks.

File: community/mpv/APKBUILD [hunk 1]
```diff
@@ -5,21 +5,49 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=mpv
 pkgver=0.29.1
-pkgrel=1
+pkgrel=2
```
Comment (Ikke) [inline community/mpv/APKBUILD]: Are there any function changes, or is it just style changes? In the case of the latter, a `pkgrel` bump is not necessary (no need to rebuild the packages)

Comment (Ikke) [reply community/mpv/APKBUILD]: Or is it necessary to rebuild because of dependencies?

Comment (tanertas) [reply community/mpv/APKBUILD]: yes, rebuild needed due to the new dependencies.

**Comment by ncopa** [comment]
I would have preferred to have each package in separate commit (but still a single PR). Would also have preferred to use `--enable-wayland` to indicate that wayland is explicitly requested, instead of relying on the autodetection.

**Comment by tanertas** [comment]
@ncopa @Ikke  can you check this again?

**Comment by tanertas** [comment]
@ncopa @ikke @andypost  I don't know why this PR closed. I just updated it recently.

**Comment by Ikke** [comment]
Because I pushed it?

**Comment by tanertas** [comment]
Ok, you can postpone this in favor of #7270 and #7272 .Those ones brings meson build system and newer libva.

**Comment by tanertas** [comment]
I removed libva* related commits. Requires #7270 and #7272.

**Comment by tanertas** [comment]
> Ok, you can postpone this in favor of #7270 and #7272 .Those ones brings meson build system and newer libva.

I updated the PR. You can remove R-main and S-hold tags. Can be committed along with #7270 and #7272.

**Pull request closed without merging**